rxbindings/src/test/java/com/amplifyframework/rx/RxStorageBindingTest.java

```diff
@@ -239,6 +288,34 @@ public void uploadFileReturnsError() {
             .assertError(error);
     }
 
+    /**
+     * When {@link StorageCategoryBehavior#uploadInputStream(String, InputStream, Consumer, Consumer)} returns
+     * an {@link StorageException}, then the {@link Single} returned by
+     * {@link RxStorageCategoryBehavior#uploadInputStream(String, InputStream)} should emit a {@link StorageException}.
+     */
+    @Test
+    public void uploadInputStreamReturnsError() {
+        StorageException error = new StorageException("Error uploading.", "Expected.");
+        doAnswer(invocation -> {
+            // 0 key, 1 local, 2 options, 3 onProgress 4 onResult, 5 onError
+            final int indexOfResultConsumer = 5;
+            Consumer<StorageException> errorConsumer = invocation.getArgument(indexOfResultConsumer);
+            errorConsumer.accept(error);
+            return mock(StorageUploadInputStreamOperation.class);
+        }).when(delegate).uploadInputStream(eq(remoteKey),
+                eq(localInputStream),
+                any(StorageUploadInputStreamOptions.class),
+                anyConsumer(),
+                anyConsumer(),
+                anyConsumer());
+
+        rxStorage
+                .uploadInputStream(remoteKey, localInputStream)
```

Comment:
Where possible, try to stick with a consistent level of indentation within a single file. Earlier, you use +4. Here, you use +8. The code base does mix the two styles, but it would be a good goal to be consistent within a single document.